src/AlphaVSS.Platform/Source/VssAsyncResult.cpp

```diff
@@ -46,6 +49,9 @@ namespace Alphaleonis { namespace Win32 { namespace Vss
 
       if (SUCCEEDED(hrResult))
       {
+		  // zDougie - something is wrong here.  m_vssAsync becomes null.
```

Comment:
The only way `m_vssAsync` should become `null` is if the finalizer has been run. This means that someone is not holding on to the object anymore. Can we open a separate issue for this with some examples on perhaps how to reproduce?